doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_rest_handler.md

```diff
@@ -139,6 +139,11 @@ It is expected UTF-8 character set is maintained by all the Flume elements in th
 
 [Top](#top)
 
+### <a name="section2.2"></a>Non valid characters
+An Attribute value of a TextUnrestricted type from an ContextBroker event is escaped from character ' to ''.
```

Comment:
```suggestion
### <a name="section2.3"></a>Quote escaping for `TextUnrestricted` attributes
Cygnus escapes from `'` to `''` in attributes of type `TextUnrestricted` with the aim of avoiding injection attacks.

Note that other attributes (i.e. with type different to `TextUnrestricted` don't need such escaping as single quote (`'`) is a [forbidden character in Context Broker](https://fiware-orion.readthedocs.io/en/master/user/forbidden_characters/index.html) so that value will never arrives to Cygnus in notifications.
```

A proper link should be also added in the ToC at the top of the file.